yoti-sdk-api/src/test/java/com/yoti/api/client/docs/session/create/filters/DocumentRestrictionsFilterTest.java

```diff
@@ -92,4 +92,14 @@ public void shouldAcceptPreBuiltDocumentRestriction() {
         assertThat(result.getDocuments(), containsInAnyOrder(documentRestrictionMock));
     }
 
+    @Test
```

Comment:
Can we add a test to ensure, that if no value is provided (i.e. the `with` method is not called), that the value passed from the builder to the domain model object is `null`?

Reply:
Added a separate test to prove this